runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -2914,7 +2914,8 @@ class INTERPRETER_CLASS
 	inlClassIsIdentity(REGISTER_ARGS_LIST)
 	{
 		J9Class *receiverClazz = J9VM_J9CLASS_FROM_HEAPCLASS(_currentThread, *(j9object_t*)_sp);
-		bool isIdentity = J9ROMCLASS_HAS_IDENTITY(receiverClazz->romClass);
+		bool isIdentity = !J9ROMCLASS_IS_VALUE(receiverClazz->romClass);
+
 		returnSingleFromINL(REGISTER_ARGS, (isIdentity ? 1 : 0), 1);
```

Comment:
Suggest updating the name to avoid the negation:
```c
		bool isValue = J9ROMCLASS_IS_VALUE(receiverClazz->romClass);

		returnSingleFromINL(REGISTER_ARGS, (isValue ? 0 : 1), 1);
```